src/api/schemas.ts

```diff
@@ -198,6 +198,22 @@ export const LimitParam = Type.Integer({
   description: 'Results per page',
 });
 
+export const Brc20OperationParam = Type.Union(
+  [
+    Type.Literal('deploy'),
+    Type.Literal('mint'),
+    Type.Literal('prepare_transfer'),
+    Type.Literal('transfer'),
```

Comment:
I think these need to be renamed because BRC-20 standard calls the actual inscritpion a `transfer`, not a prepare transfer. Perhaps better names would be `transfer` and `transfer_send` or something similar